sources/packages/web/src/views/institution/designations/ManageDesignations.vue

```diff
@@ -72,7 +72,7 @@ export default defineComponent({
           name: InstitutionRoutesConst.DESIGNATION_REQUEST,
         });
       } else {
-        snackBar.warn("There is already a pending designation agreement.");
```

Comment:
Did you check if the UI catching the message for the duplicate designation agreement? Could you add a screenshot at the PR description, please?

Reply:
I did. I have now also added the applicable screenshot.